shared/test/linkifiedTextView_test.js

```diff
@@ -197,24 +197,29 @@ describe("loop.shared.views.LinkifiedTextView", function() {
           markup: '<p><a href="http://example.com/">http://example.com/</a></p>'
         },
         {
+          desc: "should linkify a string containing only a URL with no trailing slash",
+          rawText: "http://example.com",
+          markup: '<p><a href="http://example.com/">http://example.com/</a></p>'
+        },
+        {
```

Comment:
Not redundant, tests output of http://example.com and the previous test tests http://example.com/
Making sure it adds the slash and that it does not add extra slashes..